### PR TITLE
Call the TMDB trending movie endpoint

### PR DESCRIPTION
**Pull request by RonakSharma1**
Connects to the the TMDB Api and calls the trending endpoint. This will collect this static data and then send over to react app

### CODE REVIEW

File: src/Routes/ExperimentTwo/TMDBApiCaller.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const axios = require("axios");
+var config = require("../../config");
+
+module.exports = async (req, res) => {
+  const tmdbApiKey = config.TMDB.tmdbApiKey;
```
Comment (george1410) [inline src/Routes/ExperimentTwo/TMDBApiCaller.js:5]: good call keeping the secrets in an untracked file 👍

File: src/ExpressServer.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+const express = require("express");
+const cors = require("cors");
+const boredApiRouteHandler = require("./Routes/ExperimentOne/BoredApiCaller");
+const tmdbApiRouteHandler = require("./Routes/ExperimentTwo/TMDBApiCaller");
 
 const app = express();
 
 //MiddleWare Setup
-app.use(cors());
+app.use(cors()); // Is this required to get the Keys etc
```
Comment (george1410) [inline src/ExpressServer.js:9]: not sure what this comment is meant to be saying.. but the CORS middleware is used to set some CORS headers in the response, which allows your ui to talk to a backend on a different origin.

File: src/ExpressServer.js [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 
 //Start the server
-const port=process.env.PORT || 5000
-app.listen(port,()=>{
-   console.log('Server has started on port ' + port)
-});
\ No newline at end of file
+const port = process.env.PORT || 5000;
+app.listen(port, () => {
+  console.log("Server has started on port " + port);
```
Comment (george1410) [inline src/ExpressServer.js:18]: my personal opinion but i'd use template string here
```suggestion
  console.log(`Server has started on port ${port}`);
```

**Review by george1410 — APPROVED** [review]
🚢  it

also might want to add quality gates on this repo using github actions like on the ui one

File: src/ExpressServer.js (hunk 3 above)
Comment (RonakSharma1) [reply src/ExpressServer.js:18]: Yeah I agree

File: src/ExpressServer.js (hunk 2 above)
Comment (RonakSharma1) [reply src/ExpressServer.js:9]: Yeah realised it wasn't too clear. I wanted to know is this step performed to get api-key or set up some configurations before the server actually starts working

**Pull request merged**